libnavigation-core/src/test/java/com/mapbox/navigation/core/NavigationFilesTests.kt

```diff
@@ -0,0 +1,70 @@
+package com.mapbox.navigation.core
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.mapbox.base.common.logger.Logger
+import com.mapbox.navigation.base.options.HistoryRecorderOptions
+import com.mapbox.navigation.base.options.NavigationOptions
+import com.mapbox.navigation.base.options.RoutingTilesOptions
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class NavigationFilesTests {
+
+    private val context: Context = ApplicationProvider.getApplicationContext()
+    private val navigationOptionsBuilder = NavigationOptions.Builder(context)
+    private val logger: Logger = mockk(relaxUnitFun = true)
+    private val navigationFiles = NavigationFiles(context, logger)
+
+    @Test
+    fun `history fileDirectory is default when no options provided`() {
+        val options = navigationOptionsBuilder.build()
+        val defaultDirectory = navigationFiles.historyAbsolutePath(options)
+
+        Assert.assertTrue(defaultDirectory.endsWith("mbx_nav/history"))
```

Comment:
NIT

```suggestion
        assertTrue(defaultDirectory.endsWith("mbx_nav/history"))
```

Same thing applies to the rest of the tests 👇

Reply:
done